WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1302,11 +1306,16 @@ private void toggleTagFollowStatus() {
     public void onPostSelected(long blogId, long postId) {
         if (!isAdded()) return;
 
-        AnalyticsTracker.track(AnalyticsTracker.Stat.READER_OPENED_ARTICLE);
+        ReaderPostListType type = getPostListType();
+        Map<String, Object> analyticsProperties = new HashMap<>();
+        analyticsProperties.put(READER_DETAIL_TYPE_KEY, READER_DETAIL_TYPE_NORMAL);
```

Comment:
I don't think you want to put `READER_DETAIL_TYPE_NORMAL` here since that means it's always tracked regardless of the type of detail being shown. Instead, something like this where the key is only set once based on the detail type:

```
@Override
public void onPostSelected(long blogId, long postId) {
    if (!isAdded()) return;

    ReaderPostListType type = getPostListType();
    Map<String, Object> analyticsProperties = new HashMap<>();

    switch (type) {
        case TAG_FOLLOWED:
        case TAG_PREVIEW:
            String key = (type == ReaderPostListType.TAG_PREVIEW ? READER_DETAIL_TYPE_TAG_PREVIEW : READER_DETAIL_TYPE_NORMAL);
            analyticsProperties.put(READER_DETAIL_TYPE_KEY, key);
            ReaderActivityLauncher.showReaderPostPagerForTag(
                    getActivity(),
                    getCurrentTag(),
                    getPostListType(),
                    blogId,
                    postId);
            break;
        case BLOG_PREVIEW:
            analyticsProperties.put(READER_DETAIL_TYPE_KEY, READER_DETAIL_TYPE_BLOG_PREVIEW);
            ReaderActivityLauncher.showReaderPostPagerForBlog(
                    getActivity(),
                    blogId,
                    postId);
            break;
    }
    AnalyticsTracker.track(AnalyticsTracker.Stat.READER_OPENED_ARTICLE, analyticsProperties);
}
```